imports/plugins/core/ui-tagnav/client/components/tagNav.js

```diff
@@ -0,0 +1,71 @@
+import React, { PropTypes } from "react";
+import { DragDropProvider } from "/imports/plugins/core/ui/client/providers";
+import { PropTypes as ReactionPropTypes } from "/lib/api";
+import { TagList } from "/imports/plugins/core/ui/client/components/tags/";
+
+const TagNav = (props) => {
+  return (
+    <div className={`rui tagnav ${props.navbarOrientation} ${props.navbarPosition} ${props.navbarAnchor} ${props.navbarVisibility}`}>
+      <div className="navbar-header">
+        <p>Header</p>
+      </div>
+      <DragDropProvider>
+        <TagList
+          {...props}
```

Comment:
With { ...props } you don't have to list out all the props. https://zhenyong.github.io/react/docs/jsx-spread.html